airflow/providers/postgres/hooks/postgres.py

```diff
@@ -58,7 +59,9 @@ def __init__(self, *args, **kwargs):
         self.connection = kwargs.pop("connection", None)
         self.conn = None
 
-    def _get_cursor(self, raw_cursor):
+    def _get_cursor(self, raw_cursor: str) -> Union[psycopg2.extras.DictCursor,
+                                                    psycopg2.extras.RealDictCursor,
+                                                    psycopg2.extras.NamedTupleCursor]:
```

Comment:
Would be great a type alias to the return type here

Reply:
Is it gonna be something like this:

```
    conn_name_attr = 'postgres_conn_id'
    default_conn_name = 'postgres_default'
    supports_autocommit = True
    return_cursor_type = Union[psycopg2.extras.DictCursor, 
                               psycopg2.extras.RealDictCursor, 
                               psycopg2.extras.NamedTupleCursor]
    def __init__():
    
    def _get_cursor(self, raw_cursor: str) -> return_cursor_type:
```
Does it make sense?

Reply:
Create the type alias outside of the class:
```python
_CURSOR_TYPE = Union[
    psycopg2.extras.DictCursor,
    psycopg2.extras.RealDictCursor, 
    psycopg2.extras.NamedTupleCursor
]

class Example:
    def _get_cursor(self, raw_cursor: str) -> _CURSOR_TYPE:
        ...
```